### ISSUE TITLE
Bower components included in repository

### ISSUE THREAD

**Issue by jakubzapletal** [spapali/Blue-Economics#31]
Hey guys. I see there are bower components included in git repository. It's really wrong. This way it's a pity not to use Bower (http://bower.io/) advanteges. How doesn't know it's something like Composer but for front-end packages (html, css, js, ...). It's useless to keep 3rd party packages in our repository and also we loose advantage to update these packages. A configuration file bower.js contain information about a version of all packages and it's good to know a version of package when you looking some details about it.

There should be the bower.js file in a root of project and everyone should install defined packages by command line.

Is there any volunteer to do that? Or I can do that if you wish.

@innls @zbirmingham @Mittals @greendost @jemuelyoung

**Comment by zbirmingham** [comment]
Certainly we can do that, the only reason it is included is because we were experimenting with using BigVideo.js, which has a bower components dependency.

**Comment by jakubzapletal** [comment]
Yeah I think Bower is useful for us. For example it's better to have one JS file in production so thanks to some Bower components we can use minification, concat, code quality tools, etc.

Who will implement Bower? If nobody wants, I can.

Btw.: If we implement Bower, we also need to write documentation to README for all.

**Comment by zbirmingham** [comment]
@jemuelyoung, what are your thoughts on this?

**Comment by young** [comment]
Ideally we should work towards a frontend build process that minifies the JS and CSS into singular files and lints the code. Right now we should stay focused on getting the site actually built and worry about tooling later though it's a good idea to keep these things in mind as a stretch goal.

@jakubzapletal @zbirmingham What you guys should be working on is the common CSS so that it's will be easier and faster to build out more pages.

**Comment by jakubzapletal** [comment]
@jemuelyoung yeah you are right and I agree. I have just reflected the situation with included bower components files in GitHub a repository. I wanted to mention that a better practise is to use bower.js file.
